pybossa/api/task.py

```diff
@@ -90,7 +92,10 @@ def _preprocess_post_data(self, data):
             hdfs_task = any([val.startswith("/fileproxy/hdfs/") for val in info.values() if isinstance(val, str)])
             if hdfs_task:
                 raise BadRequest("Invalid task payload. HDFS is not supported")
-        duplicate = task_repo.find_duplicate(project_id=project_id, info=info)
+        project = get_project_data(project_id)
+        dup_checksum = generate_checksum(project=project, task=data)
+        data["dup_checksum"] = dup_checksum
+        duplicate = task_repo.find_duplicate(project_id=project_id, info=info, dup_checksum=dup_checksum)
```

Comment:
It may be useful to log the checksum generated along with an indication if a duplicate record was found, for debugging purposes.

Reply:
checksum value is been logged upon generation [here](https://github.com/bloomberg/pybossa/pull/1025/files/b87460400c2713e41a12ccd539bb405505152eb5#diff-6f089ebc6d4f735099b74351d353800b7cbdf445efd97796eac09a207a58439cL235). If there's duplicate, 409 is logged by exception raised [here](https://github.com/bloomberg/pybossa/blob/main/pybossa/api/task.py#L99)

Reply:
consolidated logs to report checksum value and duplicate task id when duplicate task is found